controllers/imagejob/imagejob_controller.go

```
@@ -386,6 +419,13 @@ func (r *Reconciler) handleNewJob(ctx context.Context, imageJob *eraserv1alpha1.
 		}
 		log.Info("Started eraser pod on node", "images", imgList.Spec.Images)
 	}
+
+	imageJob.Status.Skipped = skipped
+	if err := r.updateJobStatus(ctx, imageJob); err != nil {
+		log.Error(err, "failure updating job status")
```

This should either return an error or log the error, but not both.

---

OK, we'll bubble the error up.